merkle/smt/tile_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package smt
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/google/trillian/merkle/maphasher"
+	"github.com/google/trillian/storage/tree"
+)
+
+func TestTileScan(t *testing.T) {
+	lo := tree.NewLayout([]int{8, 24})
+	h := bindHasher(maphasher.Default, 1)
+
+	ids := []tree.NodeID2{
+		tree.NewNodeID2("\x00", 8),
+		tree.NewNodeID2("\x02", 8),
+		tree.NewNodeID2("\xF0", 8),
+		tree.NewNodeID2("\x00\x01\x02\x03", 32),
+	}
+	prefs := func(high, low uint, ids ...tree.NodeID2) []tree.NodeID2 {
```

Comment:
`prefixes`? 

(prefs is quite hard-coded to 'preferences' in my old brain - _Likely reason for that:_  ![](http://guide.abime.net/wb3.1/images/smode1.gif)
)